### PR TITLE
Adding double-dot functionality to defSurv

### PR DESCRIPTION
**Pull request by kgoldfeld**
Addressing Issue #192

### CODE REVIEW

File: R/generate_data.R [hunk 1]
```diff
@@ -1035,27 +1035,38 @@ genSurv <- function(dtName, survDefs, digits = 3,
   
   for (i in (seq_along(events))) {
     
-    nlogu <--log(stats::runif(nrow(dtSurv), min = 0, max = 1))
+    nlogu <- -log(stats::runif(nrow(dtSurv), min = 0, max = 1))
```
Comment (assignUser) [inline R/generate_data.R:1038]: An important space :D

**Review by assignUser — COMMENTED** [review]
Nice!  :rocket:

File: R/generate_data.R (hunk 1 above)
Comment (kgoldfeld) [reply R/generate_data.R:1038]: @assignUser Very important. Interestingly, data.table has literally implemented its own double-dot notation that does virtually the same thing ours does. There's actually no need to parse it, data.table does it for you. But, I just used ours for consistency sake.

Comment (assignUser) [reply R/generate_data.R:1038]: Oh neat I did not know that, dt was the inspiration for the `..` prefix  :D

**Pull request merged**